editor/components/inserter/index.js

```diff
@@ -65,13 +60,7 @@ class Inserter extends Component {
 					</IconButton>
 				) }
 				renderContent={ ( { onClose } ) => {
-					const onSelect = ( item ) => {
-						onInsertBlock( item );
-
-						onClose();
-					};
-
-					return <InserterMenu items={ items } onSelect={ onSelect } rootUID={ rootUID } />;
```

Comment:
This is nice to see removed; it's always a bit of a code smell to have a component that just passes props down to other components without using them.